CONTRIBUTION.md

```diff
@@ -0,0 +1,16 @@
+# Adding simulations and experiments
+
+Adding new data is described in details in [the NMRLipids documentation](https://nmrlipids.github.io/). Please follow the instructions carefully for both simulations and experiments, filling metadata fields with meaningful and complete information.
+
+If data addition requires molecules which is currently absent, you are very wellcome to add new molecules. Please be careful with modifying molecule information, mapping files of molecules as it could affect the recomputation of other contributors' data.
```

Comment:
"If data addition requires molecules which is currently absent" -> "If data addition requires a molecule that is currently absent"